src/Environment.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App;
+
+use RuntimeException;
+
+use function in_array;
+
+final class Environment
+{
+    public const DEVELOPMENT = 'development';
+    public const TESTING = 'testing';
+    public const PRODUCTION = 'production';
+
+    private static array $values = [];
+
+    public static function prepare(): void
+    {
+        self::setEnvironment();
+        self::setBoolean('YII_C3', false);
```

Comment:
MetaStorm could be useful here :)

Reply:
This method cast value to required type. How MetaStorm will be useful in this case?